actionpack/lib/action_controller/metal/redirecting.rb

```diff
@@ -50,6 +50,9 @@ module Redirecting
     #   redirect_to post_url(@post), status: 301, flash: { updated_post_id: @post.id }
     #   redirect_to({ action: 'atom' }, alert: "Something serious happened")
     #
+    # Statements after redirect_to in our controller gets executed, so redirect_to don't stops the execution of the function.
+     # To terminate the execution of the function immediately after the redirect_to, use return.
+    #   redirect_to post_url(@post) and return
```

Comment:
Can you fix the grammar error here?
"don't stops" -> "doesn't stop"

Also for the example can you use `&&` instead of `and`?